samples/reactjs-todo/client/components/utilities/loading.js

```diff
@@ -18,15 +18,15 @@ import { AppContext } from '../../global-state';
  * @param {string} props.message - The message string object passed from the parent component
  * @returns {Object} - React component object
  */
-export default function Loading({ classes, message }) {
+export default function Loading({ classes, message, header }) {
   const [state] = useContext(AppContext);
-
   return (
     <div className="container">
       <p className={classes}>
         <span className="d-flex justify-content-center my-2">
           <span className="cstm_loading-spinner spinner-border text-primary" role="status"></span>
         </span>
+        <span className="d-flex justify-content-center fw-bolder">{header}</span>
```

Comment:
This needs a default and something that doesnt render a header when it isn't passed in. we use this component without header prop in other areas

Reply:
done